src/views/SignTransactionLedger.vue

```diff
@@ -85,18 +89,26 @@ import StatusScreen from '../components/StatusScreen.vue';
 import { Static } from '../lib/StaticStore';
 import { Getter } from 'vuex-class';
 import { State as RpcState } from '@nimiq/rpc';
-import { ParsedCheckoutRequest, ParsedSignTransactionRequest, RequestType } from '../lib/RequestTypes';
+import {
+    ParsedCashlinkRequest,
+    ParsedCheckoutRequest,
+    ParsedSignTransactionRequest,
+    RequestType,
+} from '../lib/RequestTypes';
 import { WalletInfo } from '../lib/WalletInfo';
-import { ERROR_CANCELED, TX_VALIDITY_WINDOW, CASHLINK_FUNDING_DATA } from '../lib/Constants';
+import { CASHLINK_FUNDING_DATA, ERROR_CANCELED, TX_VALIDITY_WINDOW } from '../lib/Constants';
```

Comment:
Alphabetical sorting? Is that necessary?

Reply:
My IDE does that automatically, so why not.